internal/inspect/detectors/quarto.go

```diff
@@ -181,7 +181,7 @@ func (d *QuartoDetector) InferType(base util.AbsolutePath, entrypoint util.Relat
 		// Optimization: skip inspection if there's a specified entrypoint
 		// and it's not one of ours.
 		suffix := entrypoint.Ext()
-		if suffix != ".qmd" && suffix != ".Rmd" {
+		if suffix != ".qmd" && suffix != ".Rmd" && suffix != ".ipynb" {
```

Comment:
nit: now that this is back to the full set of suffixes, this could be written as:
```
		if !slices.Contains(quartoSuffixes, entrypoint.Ext()) {
			return nil, nil
		}
```

Reply:
Nice! OK, making the change.